master/buildbot/newsfragments/fix_remoteuserauth_typestr.bugfix

```diff
@@ -0,0 +1,4 @@
+:issue:`4402`
+
+Fix RemoteUserAuth.maybeAutLogin consumes bytes object as str leading to 
```

Comment:
Could you put entire sentence in one line and then append `( :issue:`4402` )` at the end?

Reply:
Done